science/limesuite/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+name                limesuite
+categories          science comms
+platforms           darwin macosx
+license             Apache-2
+maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintainer
+description         provides drivers and SDR application support for the LMS7002M RFIC
+long_description    Lime Suite is a collection of software supporting several hardware \
+    platforms including the LimeSDR, drivers for the LMS7002M transceiver RFIC, and other \
+    tools for developing with LMS7-based hardware.
+homepage            https://myriadrf.org/projects/lime-suite/
+
+subport limesuite-devel {}
+if {[string first "-devel" $subport] > 0} {
+
+    github.setup    myriadrf LimeSuite 42f752af905a5b4464cdb95964e408a4682b4ffa
+    version         20190213
+    github.livecheck.branch master
+    checksums       rmd160  9545604b01f94ba80ebdb57908667827709f4160 \
+        sha256  58534ad960dec86ad1d5467d9259aec59634531380a7cb7549c1ca4c8b3a401d \
+        size    5347650
+
+    name            limesuite-devel
+    conflicts       limesuite
+
+} else {
+
+    github.setup    myriadrf LimeSuite 19.01.0 v
+    checksums       rmd160  6286e1ec881b5944d9913f802cba2b8236908917 \
+        sha256  bf0ca8225e9eacf44cee7383504e699eb9be5cc47293fb9c08baaf8d5e3163c9 \
+        size    5347689
+
+    conflicts       limesuite-devel
+    configure.args-append -DLIME_SUITE_EXTVER=release
+
+}
```

Comment:
I'd recommend adding in `long_description` that uses the prior version and augments it with something like:
`${long_description} This port is kept up with the ${name} GIT 'master' branch, is typically updated weekly to monthly.` ... just fair to give the user notice before using.

Reply:
Do you mean in -devel subport section?
like `long_description ${long_description} This port is kept up with the LimeSuite GIT 'master' branch, is typically updated weekly to monthly.`

Reply:
Yes: Within the "devel" subport section.